### ISSUE TITLE
The patch on postgres is selecting every row from the table before updating

### ISSUE THREAD

**Issue by andymoon** [feathersjs/feathers-sequelize#168]
I believe the line 
```
      return this._getOrFind(id, options)
```
should be 
```
      return this._getOrFind(id, params)
```
https://github.com/feathersjs/feathers-sequelize/blob/edc431f82018b7f6f376030d05225b017f26bc41/src/index.js#L180

**Comment by andymoon** [comment]
With the debug level logs I can see it make two queries against the table one for the count the other for every row. The queries have no where clauses while the update does.  I believe the first two queries are coming from the this._getOrFind function call.

**Comment by daffl** [comment]
I think that makes sense. Could you submit a PR with the change you proposed?

**Comment by andymoon** [comment]
#169

**Comment by daffl** [comment]
Closed via #169
